Framework/CurveFitting/src/TableWorkspaceDomainCreator.cpp

```diff
@@ -0,0 +1,772 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+// Includes
+//----------------------------------------------------------------------
+
+#include "MantidCurveFitting/TableWorkspaceDomainCreator.h"
+
+#include "MantidAPI/FunctionDomain1D.h"
+#include "MantidAPI/ITableWorkspace_fwd.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAPI/TableRow.h"
+#include "MantidAPI/TextAxis.h"
+#include "MantidAPI/WorkspaceFactory.h"
+#include "MantidAPI/WorkspaceProperty.h"
+
+#include "MantidCurveFitting/ExcludeRangeFinder.h"
+#include "MantidCurveFitting/Functions/Convolution.h"
+#include "MantidCurveFitting/ParameterEstimator.h"
+#include "MantidCurveFitting/SeqDomain.h"
+
+#include "MantidDataObjects/TableColumn.h"
+
+#include "MantidKernel/ArrayOrderedPairsValidator.h"
+#include "MantidKernel/ArrayProperty.h"
+#include "MantidKernel/BoundedValidator.h"
+#include "MantidKernel/EmptyValues.h"
+
+namespace Mantid {
+namespace CurveFitting {
+
+namespace {
+/**
+ * A simple implementation of Jacobian.
+ */
+class SimpleJacobian : public API::Jacobian {
+public:
+  /// Constructor
+  SimpleJacobian(size_t nData, size_t nParams)
```

Comment:
Is this the same as the implementation in `IMWDomainCreator.cpp`? If so could we move the it to a separate header/source file and include it in both to reduce the code duplication?

Reply:
Ah yes it is. I have also just noticed there already is a `Jacobian` file in which seems to do the same thing so I'm going change it to use that one.